static/js/constants.js

```diff
@@ -12,6 +12,8 @@ export const DASHBOARD_FORMAT = 'MMM D, Y';
 export const DASHBOARD_MONTH_FORMAT = 'MM[/]YYYY';
 
 export const CP1252_REGEX = /^[\u0020-\u00FF]*$/;
+// greater-than, comma, quotation-mark
+export const INVALID_NAME_CHAR_REGEX = /[\u003E\u002C\u0022]/;
```

Comment:
what about  `opening angular bracket <`. It allowed me to use it.

<img width="403" alt="screen shot 2017-03-15 at 3 10 56 pm" src="https://cloud.githubusercontent.com/assets/10431250/23943532/9ed0d140-0991-11e7-9c65-0da9475db705.png">